Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM python:3-alpine
+COPY . /app
```

Comment:
This needs a `.dockerignore` with `.git`. Otherwise you're copying `.git`. Unless it's intentional, which I don't see.

Reply:
done